packages/office-ui-fabric-react/package.json

```diff
@@ -53,7 +53,8 @@
     "@uifabric/styling": ">=5.17.0 <6.0.0",
     "@uifabric/utilities": ">=5.10.1 <6.0.0",
     "prop-types": "^15.5.10",
-    "tslib": "^1.7.1"
+    "tslib": "^1.7.1",
+    "@uifabric/variants": "*"
```

Comment:
todo: remove test code

Reply:
hmmm. This PR is massively old. Can we close it? Can you remove the todo?

Reply:
@dzearing I'm actively working on this now. I started it a long time ago since we had redlines, but hadn't fully funded the work, but now we have so I'm cleaning it up to get checked in.